connector/exceptionsconnector/README.md

```diff
@@ -3,18 +3,19 @@
 <!-- status autogenerated section -->
 | Status        |           |
 | ------------- |-----------|
-| Distributions | [] |
+| Distributions | [contrib] |
```

Comment:
Please open a PR in https://github.com/open-telemetry/opentelemetry-collector-releases to add this component to the contrib distro

Reply:
Done: https://github.com/open-telemetry/opentelemetry-collector-releases/pull/515